docs/prune.md

```diff
@@ -0,0 +1,186 @@
+# prune directive in kustomization.yaml
+
+Since Kustomize 2.1.0, a new field `prune` is added and can be used as the following in kustomization.yaml:
+```yaml
+prune:
+  type: ConfigMap
+  configMap:
+    name: prune-cm-name
+    namespace: some-namespace
+```
```

Comment:
> If `kustomize build` sees this field, it will create a `ConfigMap` object that contains a 
> condensed manifest of all _N_ resources resulting from the build process, and add that
> ConfigMap to the result as resource _N+1_ with the name as specified (in this case,
> `prune-cm-name`).

We should come up with a name for this object
* root object
* hashed manifest object

then use that name as field name instead of `prune`.

And refer to the object that way below, instead of calling it a ConfigMap

Reply:
As discussed offline, we will name it `inventory` object.